### ISSUE TITLE
Feature Request: Disable Friendly/Enemy Auras separately

### ISSUE THREAD

**Issue by Venara** [kesava-wow/kuinameplates2#220]
Hey,

Since 7.2.5 myself and other players have experienced a variety of issues tracking back to addons wanting to modify auras on friendly nameplates in dungeons/raids. The quick fix is to either disable auras or disable friendly nameplates. 

However, I'm a main healer and I use friendly nameplates to track positions of my raid members, so disabling nameplates isn't great. Similarly, if I disable auras, I don't get auras for enemy nameplates any longer. 

Now this issue is may not be originating from kuiNameplates but it eould help address the issue if you could add an option to disabble auras on friendly nameplates only (and possibly only in dungeons/raids). This way those of us affected can use the option to get rid of the issues but have the nameplates otherwise unaffected.

Thank you kindly for considering the request :)

**Comment by kesava-wow** [comment]
The nameplates causing the issue are in the restricted environment. This means add-ons aren't allowed to make changes to them, which means I can't add an option to disable auras on them. Or rather, if I did, it would make no difference to those nameplates. It'd only affect those which don't cause this problem. Same with just disabling auras entirely - that option only affects my nameplates, not the default UIs, or those in the restricted environment.

For most people, myself included, the add-on causing this problem is OmniCC - it was updated for 7.2 but the author isn't particularly active, so I assume they didn't encounter this in testing.
